content/en/tracing/trace_collection/library_injection.md

```diff
@@ -0,0 +1,749 @@
+tracing_sampling_rate: 1.0
+tracing_rate_limit: 1
+tracing_tags: 
+- a=b 
+- foo
+tracing_service_mapping:
+- from_key: mysql
+  to_name: super_db
+- from_key: postgres
+  to_name: my_pg
+tracing_agent_timeout: 1
+tracing_header_tags:
+- header: HEADER
+  tag_name: tag
+tracing_partial_flush_min_spans: 1
+tracing_debug: true
+tracing_log_level: debug
+```
+
+Set `service_language` to one of the following values:
+- `java`
+- `node`
+- `dotnet`
+
+In this configuration file, the value of `version` is always `1`. This refers to the configuration schema version in use, not the version of the content.
+
+The following table shows how the injection configuration values map to the corresponding [tracing library configuration options][2]:
+
+| Injection | Java tracer | NodeJS tracer | .NET tracer |
+| --------- | ----------- | ------------- | ----------- |
+| `tracing_enabled` | `dd.trace.enabled` | `DD_TRACE_ENABLED` | `DD_TRACE_ENABLED` |
+| `log_injection_enabled` | `dd.logs.injection` | `DD_LOGS_INJECTION` | `DD_LOGS_INJECTION` |
+| `health_metrics_enabled` | `dd.trace.health.metrics.enabled` |    n/a   |    n/a  |
+| `runtime_metrics_enabled` | `dd.jmxfetch.enabled` | `DD_RUNTIME_METRICS_ENABLED` | `DD_RUNTIME_METRICS_ENABLED` |
+| `tracing_sampling_rate` | `dd.trace.sample.rate` | `DD_TRACE_SAMPLE_RATE` | `DD_TRACE_SAMPLE_RATE` |
+| `tracing_rate_limit` | n/a       | `DD_TRACE_RATE_LIMIT` | `DD_TRACE_RATE_LIMIT` |
+| `tracing_tags` | `dd.tags` | `DD_TAGS` | `DD_TAGS` |
+| `tracing_service_mapping` | `dd.service.mapping` | `DD_SERVICE_MAPPING` | `DD_TRACE_SERVICE_MAPPING` |
+| `tracing_agent_timeout` | `dd.trace.agent.timeout` |  n/a | n/a |
+| `tracing_header_tags` | `dd.trace.header.tags` |    n/a    | `DD_TRACE_HEADER_TAGS` |
+| `tracing_partial_flush_min_spans` | `dd.trace.partial.flush.min.spans` | `DD_TRACE_PARTIAL_FLUSH_MIN_SPANS` | `DD_TRACE_PARTIAL_FLUSH_ENABLED ` |
+| `tracing_debug` | `dd.trace.debug` | `DD_TRACE_DEBUG` | `DD_TRACE_DEBUG` |
+| `tracing_log_level` | `datadog.slf4j.simpleLogger.defaultLogLevel` | `DD_TRACE_LOG_LEVEL` |   n/a    |
+
+Tracer library configuration options that aren't mentioned in the injection configuration are still available for use through properties or environment variables the usual way.
+
+## Launch your services
+
+Launch your services, indicating the preload library configuration in the launch command:
+
+**Java app example**:
+```sh
+DD_CONFIG_SOURCES=BASIC java -jar <SERVICE_1>.jar &
+DD_CONFIG_SOURCES=LOCAL:/etc/<SERVICE_2>/config.yaml;BASIC java -jar <SERVICE_2>.jar &
+```
+
+**Node app example**:
+```sh
+DD_CONFIG_SOURCES=BASIC node index.js &
+DD_CONFIG_SOURCES=LOCAL:/etc/<SERVICE_2>/config.yaml;BASIC node index.js &
+```
+
+**.NET app example**:
+```sh
+DD_CONFIG_SOURCES=BASIC dotnet <SERVICE_1>.dll &
+DD_CONFIG_SOURCES=LOCAL:/etc/<SERVICE_2>/config.yaml;BASIC dotnet <SERVICE_2>.dll &
+```
+
+Exercise your application to start generating telemetry data, which you can see as [traces in APM][4].
```

Comment:
```suggestion
Execute your application to start generating telemetry data, which you can see as [traces in APM][4].
```

assuming "execute" is meant? unless exercising an application is also a thing

Reply:
@cswatt Yeah, exercise means "not just run it but also do some stuff in it that causes traces to happen."